honeycomb/honeycomb.go

```diff
@@ -29,81 +31,272 @@ import (
 )
 
 const (
-	defaultApiKey  = "apikey-placeholder"
 	defaultDataset = "opentelemetry"
 )
 
+// Config defines the basic configuration for the Honeycomb exporter.
 type Config struct {
-	// ApiKey is your Honeycomb authentication token, available from
-	// https://ui.honeycomb.io/account. default: apikey-placeholder
-	ApiKey string
-	// Dataset is the name of the Honeycomb dataset to which events will be
-	// sent. default: beeline-go
-	Dataset string
-	// Service Name identifies your application. While optional, setting this
-	// field is extremely valuable when you instrument multiple services. If set
-	// it will be added to all events as `service_name`
-	ServiceName string
-	// Debug will emit verbose logging to STDOUT when true. If you're having
-	// trouble getting the beeline to work, set this to true in a dev
-	// environment.
-	Debug bool
-	// APIHost is the hostname for the Honeycomb API server to which to send
-	// these events. default: https://api.honeycomb.io/
-	APIHost string
```

Comment:
I would like to keep APIHost for internal testing purposes.

Reply:
It's still there, as the `WithAPIURL` function.

Reply:
Whoops. My eyes skipped that. 👍